include/json/config.h

```diff
@@ -118,6 +120,12 @@
 #  define JSON_USE_INT64_DOUBLE_CONVERSION 1
 #endif
 
+// If non-zero, the library zeroes any memory that it has allocated before
+// it frees its
+#ifndef JSON_USE_SECURE_MEMORY
+#define JSON_USE_SECURE_MEMORY 1
```

Comment:
We'll have to check whether the typedefs alter the symbols. If so, then we'll need to switch those all to macros (poor-man's template). Again, that's to ensure binary-compatibility. It's painful, but our unusual care for binary-compatibility is one reason why a distro like Debain includes JsonCpp.